src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -707,9 +710,16 @@ public void remove() {
             scoreboard.unsubscribe(this);
             scoreboard = null;
         }
+        clearBossBars();
         super.remove();
     }
 
+    private void clearBossBars() {
```

Comment:
This method should go after the `remove(boolean async)` method.

Reply:
Done.